test/e2e/testing-manifests/kubectl/nginx-deployment1.yaml.in

```diff
@@ -1,9 +1,12 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1beta2
```

Comment:
are any of these used in skewed server tests?

Reply:
kubectl skew tests will test 1.8 kubectl against 1.7 server.